lib/double_bag_ftps.rb

```diff
@@ -109,6 +109,10 @@ def transfercmd(cmd, rest_offset = nil)
       conn = ssl_socket(conn) # SSL connection now possible after cmd sent
     else
       sock = makeport
+      # Before ruby-2.3.0, makeport did a sendport automatically
+      if RUBY_VERSION >= "2.3"
```

Comment:
Good catch.  Indeed, 2.1.0 and 2.1.1 are not affected, but 2.1.2 and 2.1.3 are.

This is why I hate explicit version checks.  That said, until I figure out how to do this without a version check, I'll add the appropriate versions of 2.1 to this _if_.